packages/manifest-helper/src/utils/error.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
is the disable lint still needed?

Reply:
It's weird, I removed this annotation yesterday. 

I may have forgotten to push it 😕 🤣